cfn-resources/cloud-backup-schedule/mongodb-atlas-cloudbackupschedule.json

```diff
@@ -1,279 +1,263 @@
+      "type": "array",
+      "insertionOrder": false,
+      "description": "List that contains a document for each copy setting item in the desired backup policy.",
+      "items": {
+        "$ref": "#/definitions/ApiAtlasDiskBackupCopySettingView",
+        "type": "object"
+      }
+    },
+    "DeleteCopiedBackups": {
+      "type": "array",
+      "insertionOrder": false,
+      "description": "List that contains a document for each deleted copy setting whose backup copies you want to delete.",
+      "items": {
+        "$ref": "#/definitions/ApiDeleteCopiedBackupsView",
+        "type": "object"
+      }
+    },
+    "Policies": {
+      "type": "array",
+      "insertionOrder": false,
+      "description": "Rules set for this backup schedule.",
+      "items": {
+        "$ref": "#/definitions/ApiPolicyView",
+        "type": "array"
+      }
+    },
+    "ReferenceHourOfDay": {
+      "description": "UTC Hour of day between 0 and 23 representing which hour of the day that Atlas takes a snapshot",
+      "type": "integer"
+    },
+    "ReferenceMinuteOfHour": {
+      "description": "UTC Minute of day between 0 and 59 representing which minute of the referenceHourOfDay that Atlas takes the snapshot.",
+      "type": "integer"
+    },
+    "RestoreWindowDays": {
+      "description": "Number of days back in time you can restore to with Continuous Cloud Backup accuracy. Must be a positive, non-zero integer.",
+      "type": "integer"
+    },
+    "UpdateSnapshots": {
+      "description": "Flag indicating if updates to retention in the backup policy were applied to snapshots that Atlas took earlier. ",
+      "type": "boolean"
+    },
+    "ClusterId": {
+      "description": "Unique identifier of the Atlas cluster.",
+      "type": "string"
+    },
+    "NextSnapshot": {
+      "description": "Timestamp in the number of seconds that have elapsed since the UNIX epoc when Atlas takes the next snapshot.",
+      "type": "string"
+    },
+    "Profile": {
+      "type": "string",
+      "description": "Profile used to provide credentials information, (a secret with the cfn/atlas/profile/{Profile}, is required), if not provided default is used",
+      "default": "default"
+    },
+    "Links": {
+      "type": "array",
+      "insertionOrder": false,
+      "description": "List of one or more Uniform Resource Locators (URLs) that point to API sub-resources, related API resources, or both. RFC 5988 outlines these relationships.",
+      "items": {
+        "$ref": "#/definitions/Link",
+        "type": "object"
+      }
+    }
+  },
+  "additionalProperties": false,
+  "required": [
+    "Profile"
+  ],
+  "writeOnlyProperties": [
+    "/properties/UpdateSnapshots",
+    "/properties/DeleteCopiedBackups"
+  ],
+  "readOnlyProperties": [
+    "/properties/ClusterId",
+    "/properties/NextSnapshot",
+    "/properties/Policies/*/PolicyItems/*/ID",
+    "/properties/ProjectId",
+    "/properties/ClusterName",
+    "/properties/Links"
```

Comment:
This is the main change

Reply:
@andreaangiolillo ProjectId would be input by the user. For now manually fixing it on cdk side.

Reply:
fixing it here https://github.com/mongodb/mongodbatlas-cloudformation-resources/pull/414